packs/data/blood-lords-bestiary.db/pesgahi-the-poisoner.json

```diff
@@ -992,15 +992,17 @@
                             "success"
                         ],
                         "selector": "{item|_id}-damage",
-                        "text": "PF2E.BombNotes.AcidFlask.Moderate.success"
+                        "text": "PF2E.BombNotes.AcidFlask.Moderate.success",
+                        "title": ""
                     },
                     {
                         "key": "Note",
                         "outcome": [
                             "criticalSuccess"
                         ],
                         "selector": "{item|_id}-damage",
-                        "text": "PF2E.BombNotes.AcidFlask.Moderate.criticalSuccess"
+                        "text": "PF2E.BombNotes.AcidFlask.Moderate.criticalSuccess",
+                        "title": ""
```

Comment:
Why a blank title?

Reply:
i don't know what that is so it might have been auto added somehow?

Reply:
Make it `{item|name}`

Reply:
added the `{item|name}`s to the titles